.drone.star

```diff
@@ -1598,7 +1598,8 @@ def makeGoGenerate(module):
             "name": "generate go",
             "image": OC_CI_GOLANG,
             "commands": [
-                "%s ci-go-generate" % (make),
+                "curl -SsfL -o /usr/local/bin/retry https://raw.githubusercontent.com/owncloud-ci/retry/master/retry && chmod +x /usr/local/bin/retry",
```

Comment:
golang images have "retry" in them now - https://github.com/owncloud-ci/golang/pull/91
So this line should not be needed.

Reply:
Removed!